OnProfileUpdate/function.json

```diff
@@ -0,0 +1,16 @@
+{
+  "bindings": [
+    {
+      "type": "cosmosDBTrigger",
+      "name": "documents",
+      "direction": "in",
+      "connectionStringSetting": "CosmosDbConnectionString",
+      "databaseName": "%COSMOSDB_NAME%",
+      "collectionName": "profiles",
+      "startFromBeginning": true,
+      "createLeaseCollectionIfNotExists": true,
+      "leaseCollectionName": "profile-emails-leases"
```

Comment:
I suggest to add a lease prefix. In this way you can put all leases in the same collection and reduce costs.

Reply:
the name with the leases suffix was similar to the other leases collections